### PR TITLE
 suggested locale fixes

### PR DESCRIPTION
**Pull request by stroucki**
Copied from pull request #226. Sorry about the mess. Also includes a new spelling fix for Spanish.

### CODE REVIEW

**Comment by strk** [comment]
LGTM -- @stroucki note that you can `git push -f fork prbranch` if you need to override a messed up PR :)

File: conf/locale/locale_de-DE.ini [hunk 1]
```diff
@@ -915,9 +915,9 @@ dashboard.current_goroutine=Aktuelle Goroutines
 dashboard.current_memory_usage=Aktuelle Speichernutzung
 dashboard.total_memory_allocated=Zugeteilter Gesamtspeicher
 dashboard.memory_obtained=Erhaltener Speicher
-dashboard.pointer_lookup_times=Pointer Lookup Times
-dashboard.memory_allocate_times=Memory Allocate Times
-dashboard.memory_free_times=Memory Free Times
+dashboard.pointer_lookup_times=Zeigerlookup-Dauern
```
Comment (tboerger) [inline conf/locale/locale_de-DE.ini]: Dauern still sounds wrong to me as a German

Comment (stroucki) [reply conf/locale/locale_de-DE.ini]: I am German too, and it is unusual in common language, but Duden says it is used in technical context.

Note that I've not seen where this text is displayed, so if it is only a single value that is displayed, it should definitely be "Dauer".

Comment (stroucki) [reply conf/locale/locale_de-DE.ini]: @strk tried but unsuccessful

Comment (tboerger) [reply conf/locale/locale_de-DE.ini]: It's a key for a table in the admin ui where you can see the garbage collection interval and so on. So it should be Dauer

Comment (stroucki) [reply conf/locale/locale_de-DE.ini]: So the English appears to be incorrect too. I'll change to "Dauer".

**Comment by tboerger** [comment]
LGTM

**Comment by tboerger** [comment]
@stroucki just for the records, this is how it looks like:

bildschirmfoto 2016-11-30 um 11 38 08

**Comment by stroucki** [comment]
Ah damn, then the proper translation should be "Anzahl", since these are counts and not clock times.

**Pull request merged**